src/Apps/Order2/Utils/errors.ts

```diff
@@ -0,0 +1,21 @@
+export { CheckoutMutationError } from "Apps/Order/Components/ExpressCheckout/Util/mutationHandling"
+
+type ExchangeErrorCode =
+  | "missing_region"
+  | "missing_country"
+  | "missing_postal_code"
+  | "destination_could_not_be_geocoded"
+
+type LocalErrorCode = "no_shipping_options"
```

Comment:
We can build these up more as we go - I think it makes matching a little easier.